src/main/java/org/terasology/genome/genomeMap/GeneIndexGenomeMap.java

```diff
@@ -16,37 +16,54 @@
 package org.terasology.genome.genomeMap;
 
 import com.google.common.base.Function;
+import org.terasology.genome.breed.BreedingAlgorithm;
+import org.terasology.genome.breed.DiploidBreedingAlgorithm;
+import org.terasology.genome.breed.MonoploidBreedingAlgorithm;
 
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 /**
  * An implementation of GenomeMap.
  */
 public class GeneIndexGenomeMap implements GenomeMap {
-    private Map<String, GenePropertyDefinition> propertyDefinitionMap = new HashMap<>();
+    public Map<String, GenePropertyDefinition> propertyDefinitionMap = new LinkedHashMap<>();
```

Comment:
Why was this made public? This allows to modify or completely swap out the map, although we give specific acccessor methods below...

As this is public, it should also have Javadoc (which would be a good place to note down why can/should be done with it).

Reply:
It was made public so the Genome Manager can access the propertyDefinitionMap to get the appropriate traits in the order they are stored and access the details. It could be kept private and use a getter method to get the Map instead.